### DISCUSSION TITLE
Can we merge https://github.com/tiangolo/fastapi/pull/5077 ?

### DISCUSSION THREAD

**Discussion by dsinghvi** [fastapi/fastapi#5781] [Questions] [answered]
Hey! Would be great if we could get this PR merged in. It's unclear what we can do to get this in, so wanted to start a larger discussion if folks had ideas?

**Comment by iudeen** [comment]
It's labeled as investigate. Which means the maintainer has marked it to be evaluated further by himself. So it's best we wait for it to happen 😌

**Comment by tiangolo** [comment, marked as answer]
Hey there, thanks for the interest.

There are lots of questions to answer and PRs to review. As answering questions (like this one) takes quite a lot of time, if you are interested in getting things done faster, you can help me answering those questions: https://fastapi.tiangolo.com/help-fastapi/#help-others-with-questions-in-github

You can also help me review PRs following all the procedure (otherwise, just an approval in the PR doesn't really help much): https://fastapi.tiangolo.com/help-fastapi/#review-pull-requests

Unfortunately, asking for the state of a PR in a new discussion doesn't really help move forward the PR, as it adds one more thing I have to review. I know it might be counterintuitive, but adding more comments asking for the state of something doesn't really help much, if anything it sadly makes things a bit slower. But if you help me answering questions from others (like @iudeen) that will help a lot, and you can become a FastAPI Expert: https://fastapi.tiangolo.com/fastapi-people/#experts
